go/vt/worker/row_splitter.go

```diff
@@ -71,7 +71,7 @@ func (rs *RowSplitter) Split(result [][][]sqltypes.Value, rows [][]sqltypes.Valu
 }
 
 // Send will send the rows to the list of channels. Returns true if aborted.
-func (rs *RowSplitter) Send(fields []mproto.Field, result [][][]sqltypes.Value, baseCmd string, insertChannels [][]chan string, abort chan struct{}) bool {
+func (rs *RowSplitter) Send(fields []mproto.Field, result [][][]sqltypes.Value, baseCmd string, insertChannels [][]chan string, abort <-chan struct{}) bool {
```

Comment:
Send doesn't actually need a context. All it needs is a channel that signifies that it shouldn't send anymore.

I find it clearer to just pass in the channel instead of assuming it does something meaningful with a context.